### ISSUE TITLE
rate your music logo

### ISSUE THREAD

**Issue by lxjv** [neatnik/omg.lol#162]
proflile link look like https://rateyourmusic.com/~laker

**Comment by newbold** [comment]
This one’s tricky! The icon system relies on single-path, monochromatic icons for compatibility with Font Awesome. Looks like RYM’s icon is a really beautiful multi-toned blue blobby thing which I can’t make render as an all black path (or all white in dark mode) which is a bummer. I’ve run into a few of these now, so I’m starting to think that the answer is for me to expand the profile icon system so that it can use PNGs and try to break out of the Font Awesome mold. What do you think?

**Comment by lakerjv** [comment]
thats alright with me! do whatever it'll need to make it work XD

**Comment by newbold** [comment]
Last year I had thought that I'd be able to run an expanded icon system that pulls in image-based icons alongside the single-path SVG icons that I use (following Font Awesome’s and other typical icon format conventions). I spent a couple of hours last night experimenting with this, and the result was... pretty bad. Turns out when you mix vector icons and bitmaps, you wind up with this jarring distinction and the bitmaps look totally _whack_ — fully out of place. And then what's worse is when you style the vectors with theming, or invert for dark/light mode, etc., the bitmaps remain unchanged and in some cases you wind up with readability and recognition issues. I was optimistic about it, but after actually trying it out, it was clear that it wasn’t going to work.

So, I think I’m going to have to throw in the towel on this. I did check on Rate Your Music’s logo to see if there had been any updates, but they’re still running with their PNG (and no one out there has attempted to create a single-path vector equivalent, that I could find, though that makes sense given that it would not be easy to do _at all_). In this case, I think the best approach is to use a custom tag like `/music/` or `/record-vinyl/` (soon to be `{music}`/`record-vinyl` in v2). I know it’s not as ideal as a real logo though. Maybe RYM will eventually follow the trend and make their own icon-compatible vector logo? We can hope!
